faheem/infra/requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e .
+aws-cdk.core==1.122.0
+aws-cdk.aws_lambda==1.122.0
+aws-cdk.aws_iam==1.122.0
```

Comment:
As aws_cdk.aws_iam is not used in the code, what do you say of removing that from requirements.txt file.

Reply:
There was an issue earlier with creating a role, that's why it was created